docs/command-line-interface/account.md

```diff
@@ -40,9 +40,10 @@ Keep your locked Gold more secure by authorizing alternative keys to be used for
 
 ```
 USAGE
-  $ celocli account:authorize --from <value> -r vote|validator|attestation --signature
-    <value> --signer <value> [--gasCurrency <value>] [--globalHelp] [--blsKey <value>
-    --blsPop <value>]
+  $ celocli account:authorize --from 0xc1912fEE45d61C87Cc5EA59DaE31190FFFFf232d -r
```

Comment:
what's the rationale for these addresses instead of the <value> placeholders?

Reply:
this is autogenerated

Reply:
ack, I'm not a fan because it's potentially confusing, but not opposed strongly either